downstream/modules/analytics/proc-link-plan-job-template.adoc

```diff
@@ -9,6 +9,6 @@
 You can associate a job template to a savings plan to allow {InsightsShort} to provide a more accurate time and cost savings estimate for completing this savings plan.
 
 .Procedure
-. Navigate to *Red Hat Insights* > *Savings Planner*.
-. Click image:images/ellipsis.png[More,10,25] > *Link Template*.
-. Click *Save*.
+. Navigate to menu:Red Hat Insights[Savings Planner].
+. Click menu:image:images/ellipsis.png[More,10,25][Link Template].
```

Comment:
Same as above, suggest changing to:

On the automation savings plan, click image:images/ellipsis.png[More,10,25] and select * Link Template*.